src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1

```diff
@@ -390,7 +390,7 @@ class SVTResourceResolver: AzSKRoot {
             }
         }
         if (-not $this.skipOrgUserControls) {
-            if ($this.ResourceTypeName -in ([ResourceTypeName]::User, [ResourceTypeName]::All, [ResourceTypeName]::Org_Project_User, [ResourceTypeName]::Build_Release_SvcConn_AgentPool_VarGroup_User)) {
```

Comment:
"Build_Release_SvcConn_AgentPool_VarGroup_User" this resource type is being use in extension also, plz check extension code and do changes there also if needed.